hebi_a-2084-01-parallel-gripper_moveit_config/config/A-2084-01.srdf

```diff
@@ -27,12 +28,7 @@
         <link name="end_effector/r_gripper" />
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="home" group="hebi_arm">
-        <joint name="HEBI/base/X5_4" value="-1" />
-        <joint name="HEBI/elbow/X5_4" value="-2" />
-        <joint name="HEBI/shoulder/X5_9" value="0" />
-        <joint name="HEBI/wrist1/X5_1" value="0" />
-    </group_state>
+    <group_state name="home" group="hebi_arm" />
```

Comment:
I'm surprised this disappeared, but kept the `group_state` tag?

I think we still need the joint angles here (and on other .srdfs in the PR) so that some of the example moveit scripts (e.g., clearpath integration) work